src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommandRunner.cs

```diff
@@ -40,7 +40,7 @@ public async Task ExecuteCommandAsync(ListPackageArgs listPackageArgs)
             }
             //If the given file is a solution, get the list of projects
             //If not, then it's a project, which is put in a list
-            var projectsPaths = Path.GetExtension(listPackageArgs.Path).Equals(".sln") ?
+            var projectsPaths = Path.GetExtension(listPackageArgs.Path).Equals(".sln", StringComparison.Ordinal) ?
```

Comment:
This looks like a path, let's make it File-System aware:

```suggestion
            var projectsPaths = Path.GetExtension(listPackageArgs.Path).Equals(".sln", PathUtil.GetStringComparisonBasedOnOS()) ?
```